hummingbird/ml/_container.py

```diff
@@ -223,6 +217,48 @@ def score_samples(self, *inputs):
         return self.decision_function(*inputs) + self._extra_config[constants.OFFSET]
 
 
+# PyTorch containers.
```

Comment:
Next time it would be really helpful if a big refactoring like this was in a separate PR. It took me a while to understand how all of this finally relates to TVM with `class TVMSklearnContainer(SklearnContainer)` 

(Ex: do this refactoring, merge that, then add in the TVM stuff. It would be cleaner for all to understand looking back if we run into issues. :) )

Reply:
I know...I just had to do this to simplify otherwise we were having a lot of duplicate code.

Reply:
No it's cool. I'll let this one slide haha :)